### DISCUSSION TITLE
[Proposal] Use the colour-picker macro more in tiddlywiki's UI

### DISCUSSION THREAD

**Discussion by Leilei332** [TiddlyWiki/TiddlyWiki5#8603] [Ideas]
Currently the colour-picker macro is only used in the image editor. IMO the colour-picker macro should replace the HTML native color picker in some UI elements, like the TagManager and the color field editor.

The advantage of the colour-picker macro is that it provides a full set of defaut css color names for us to choose when we don't know what color to use. This can make up the problem of native HTML color picker, which requires picking one color manually.

**Comment by Jermolene** [comment]
Thanks @Leilei332 I agree, that would make a lot of sense.
